app/views/stages/_fields.html.erb

```diff
@@ -28,7 +28,7 @@
       <%= form.label :name, 'Deploy Groups', class: 'col-lg-2 control-label' %>
       <div class="col-lg-4">
         <%= form.collection_select(
```

Comment:
As a rule of thumb, I dislike seeing DB queries like `DeployGroup.includes(:environment).order(:environment_id, :name).all` in the view code.